attributes/defaults.rb

```diff
@@ -18,17 +18,23 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-# Attribute that defines whether MariaDB or MySQL should be used
-default['mysqld']['use_mariadb'] = true
+# Attribute that defines whether MariaDB, MySQL or Percona should be used
+default['mysqld']['db_install'] = 'mariadb'
```

Comment:
Im seeing right now, the repository configuration must be included inside install to avoid asking the user to include the repository configuration, with percona this is necessary but i have my doubts with mariadb.